packages/games/src/constants/games/index.ts

```diff
@@ -0,0 +1,11 @@
+import shuffle from 'lodash/shuffle'
+import { GameType } from '../../types'
+// Project
+import { example } from './example'
+import { binaryX } from './binaryX'
+
+export const TOP_GAMES_LIST: GameType[] = []
+
+export const DEFAULT_GAMES_LIST: GameType[] = [binaryX, example]
+
+export const GAMES_LIST = [...TOP_GAMES_LIST, ...shuffle(DEFAULT_GAMES_LIST)]
```

Comment:
any reason for the shuffle here?

Reply:
For me, I just want make it every time random.
So every project have a chance to let user see it.
If we really want it show in front, we can move it to `TOP_GAMES_LIST`